consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/PreparedRoundMetadata.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.consensus.qbft.payload;
+
+import org.hyperledger.besu.ethereum.core.Hash;
+import org.hyperledger.besu.ethereum.rlp.RLPInput;
+import org.hyperledger.besu.ethereum.rlp.RLPOutput;
+
+import java.util.Objects;
+
+import com.google.common.base.MoreObjects;
+
+public class PreparedRoundMetadata {
+  private final Hash preparedBlockHash;
+  private final int preparedRound;
+
+  public PreparedRoundMetadata(final Hash preparedBlockHash, final int preparedRound) {
+    this.preparedBlockHash = preparedBlockHash;
+    this.preparedRound = preparedRound;
+  }
+
+  public Hash getPreparedBlockHash() {
+    return preparedBlockHash;
+  }
+
+  public int getPreparedRound() {
+    return preparedRound;
+  }
+
+  /**
+   * Constructor that derives the sequence and round information from an RLP encoded message
+   *
+   * @param in The RLP body of the message to check
+   * @return A derived sequence and round number
```

Comment:
This doc seems incorrect, we aren't returning a (height, round)

Reply:
we can go flatter - done.